src/api/services/userService.ts

```diff
@@ -34,7 +34,7 @@ export class UserService {
       id: req.user!.participantId,
     });
     const currentSite =
-      currentParticipant?.siteId === undefined
+      currentParticipant?.siteId === undefined || currentParticipant.siteId === null
```

Comment:
nice! I think we can just simplify this to `!currentParticipant?.siteId` - that checks for [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

Reply:
Ah yeah cool we could. Have made that change!